### DISCUSSION TITLE
UnityMirror Sample

### DISCUSSION THREAD

**Discussion by calebsmith12890** [PlayFab/MpsSamples#44] [Q&A] [answered]
I'm dead set on trying to get a project going with Unity/Mirror/PlayFab that has the Client and Servers in separate projects, like you have in this sample. When I download and run these two samples everything seems to connect just fine so long as I just use the Server and Client scenes. Where I'm getting confused is the implementation of the Online/Offline/Room scenes. What exactly was the intention with these scenes, are they finished, how are they supposed to be setup? It seems like I've tried every combination of using the scenes in the client and/or server network managers and the only way I can seem to get the online scene to trigger is by hosting in the single build. I hope this question makes any sense. I'd love a bit more guidance or work on this sample if possible. Thank you!

**Comment by dgkanatsios** [comment, marked as answer]
@calebsmith12890, apologies for the inconvenience, these scenes should not be there, I totally missed them when I was rebuilding the sample. The only scene that we use for the sample is the ServerScene. Hope this clears the confusion!

**Reply by calebsmith12890** [reply]
Bummer, I was excited to see those. Thought it would be really helpful if they were useable. Thanks for the response anyway!
